packages/google-calendar/src/resources/calendars/acl.ts

```
@@ -0,0 +1,786 @@
+
+  /**
+   * Body param: The type of delivery mechanism used for this channel. Valid values
+   * are "web_hook" (or "webhook"). Both values refer to a channel where Http
+   * requests are used to deliver messages.
+   */
+  type?: string;
+}
+
+export declare namespace ACL {
+  export {
+    type ACLRule as ACLRule,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)